src/js/components/uploadFabsFile/UploadFabsFileValidation.jsx

```diff
@@ -146,6 +146,10 @@ const UploadFabsFileValidation = (props) => {
         }
     }, [signInProgress]);
 
+    useEffect(() => {
+        return () => window.clearInterval(dataTimer.current);
+    }, []);
```

Comment:
Why isn't this just also part of the return for the first useEffect? Also, should there be a check to make sure a dataTimer.current exists first?